lib/chainweb/util/network.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Kadena.Chainweb.Network do
```

Comment:
@santiagoboterokommit please, remove this module since it is not related to the HTTP client impl.